packages/react-widgets/src/widgets/FormulaWidget.js

```diff
@@ -31,28 +32,35 @@ function FormulaWidget({
   joinOperation,
   formatter,
   animation,
+  global,
   onError,
   wrapperProps
 }) {
   const isSourceReady = useSelector((state) =>
     selectAreFeaturesReadyForSource(state, dataSource)
   );
-  const { filters, filtersLogicalOperator } = useSourceFilters({ dataSource, id });
+  const source = useWidgetSource({ dataSource, id });
 
   const [formulaData, setFormulaData] = useState(null);
   const [isLoading, setIsLoading] = useState(true);
 
+  const spatialFilter = useSelector((state) =>
+    global ? selectSpatialFilter(state, source?.id) : null
```

Comment:
Are we sure we want spatialFilter for global?

Reply:
This is my only 'big doubt' in PR, @Clebal

Reply:
I talked with Alberto, and he decided to remove spatial filter on global for now.

Reply:
makes sense